pact/coin-contract/coin-sig.pact

```diff
@@ -6,14 +6,19 @@
   \transfer function, coinbase, account creation and balance query."
 
   (defun create-account:string (account:string guard:guard)
-    @doc "Create an account for ACCOUNT, with ACCOUNT as a function of GUARD")
+    @doc "Create an account for ACCOUNT, with ACCOUNT as a function of GUARD"
+    @model [ (property (not (= account ""))) ]
+    )
 
-  (defun transfer:string (sender:string reciever:string receiver-guard:guard amount:decimal)
+  (defun transfer:string (sender:string receiver:string receiver-guard:guard amount:decimal)
     @doc "Transfer between accounts SENDER and RECEIVER on the same chain.    \
     \This fails if both accounts do not exist. Create-on-transfer can be      \
     \handled by sending in a create command in the same tx."
 
-    @model [ (property (> amount 0.0)) ])
+    @model [ (property (> amount 0.0))
+             (property (not (= sender reciever)))
```

Comment:
Variable name was changed, but use-site wasn't.

Reply:
Interestingly, `flycheck` isn't picking this up.

Reply:
```
colin@yumi ~/c/h/c/p/coin-contract> pact coin-sig.pact 
Load successful
```
Despite the `reciever` symbol not being in scope.

Reply:
yikes. I think analysis might be broken

Reply:
I looked into this a bit more. Analysis actually does fail on this file (for the wrong reason, which I'll fix). But when invoked this way (with no flags, on a `.pact` file), Pact runs in "compile only" mode, meaning that analysis is never even called.